commands.yaml

```diff
@@ -1,39 +1,42 @@
 help:
```

Comment:
So far I looked only at command names. I'll hold off with reviewing the new options until we have a clearer reasoning behind the shorts.